### ISSUE TITLE
Switch to PackageReference

### ISSUE THREAD

**Issue by mattleibow** [aspose-words/Aspose.Words-for-.NET#178]
HI folks, great library! Just wanted to let you know that you can switch to the new `<PackageReference>` style of importing packages into projects: https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files

There is an easy way to do this via the migration tool: https://docs.microsoft.com/en-us/nuget/reference/migrate-packages-config-to-package-reference

The main advantage is that the packages are no longer downloaded to the solution folder - both using unnecessary disk space and allowing .csproj files to be opened directly.

The reason for this issue is that I got a message on NuGet.org when a customer tried to run the sample by opening the CSharp.csproj file directly and NuGet failed to restore the files to the correct path.

Let me know if I can help in any way.

PS: Thanks for making use of SkiaSharp, I love working on that library and making improvements. If you have any suggestions, feel free to just open a new issue in the SkiaSharp repo!

**Comment by muhammad-ijaz** [comment]
@mattleibow ,

Thanks for your feedback and suggestion. We will discuss the benefits and limitations of PackageReference internally and then will migrate to it . We will keep you updated on it.

**Comment by mattleibow** [comment]
Awesome! One possible limitation I thought of is that it _may_ not work in VS2015 and below. But, you might have to check to confirm - not sure what your minimum version you need to maintain.

**Comment by muhammad-ijaz** [comment]
@mattleibow,

Yes, you are absolutely correct. This is the major limitation we are analyzing.

**Comment by romeokromeok** [comment]
@AlexNosk please check if this issue can be closed.
